pyproject.toml

```diff
@@ -118,6 +118,9 @@ doc = [
     "mkdocs-material[imaging]",
```

Comment:
Locally, I finally got this to work by updating the `mkdocs-material[imaging]` version string to `~=9.5.17` (which installed version `9.5.18` using python 3.11 conda environment + `flit install --deps develop`).

I think that likely fixed & later versions will be installed for mkdocs plugins for fresh installs of orchestrator-core local dev environment, but wanted to make a note here about this issue, which was causing

```python
zipfile.BadZipFile: File is not a zip file
```

errors I think because of some Google CDN/Font file changes.

[Here's a link to relevant "social" plugin issue on Github](https://github.com/squidfunk/mkdocs-material/issues/6983)

Reply:
Good call. I think that you will only see this if you installed `mkdocs-material` after the bug was introduced, but before it was fixed, which is a ~1 week window, right? Or do you still see this with a fresh install?

Reply:
Yes, it was a pretty narrow window! My intent was to make a note in case anyone else ran into it, however unlikely. Gonna resolve the the comment though

Reply:
Really is a shame that flit doesn't have an upgrade option. When testing this, the only real workaround I could find was to upgrade with pip. But my skill level on these things is not great 😅